bundle/manifests/ingress-node-firewall.clusterserviceversion.yaml

```diff
@@ -239,14 +239,14 @@ spec:
                 - /manager
                 env:
                 - name: DAEMONSET_IMAGE
-                  value: quay.io/mmahmoud/ingress-node-firewall-daemon:latest
+                  value: quay.io/openshift/origin-ingress-node-firewall-daemon:latest
                 - name: DAEMONSET_NAMESPACE
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
                 - name: KUBE_RBAC_PROXY_IMAGE
                   value: quay.io/openshift/origin-kube-rbac-proxy:latest
-                image: quay.io/mmahmoud/controller:latest
+                image: quay.io/mmahmoud/ingress-node-firewall-controller:latest
```

Comment:
Can't this be updated to use `openshift` repo?

Reply:
ofcourse I thought I did that not sure why it flipped back anyway I fixed both comments Thanks